### DISCUSSION TITLE
ImportError: DLL load failed while importing CSXCAD: 找不到指定的模块。

### DISCUSSION THREAD

**Discussion by Derek-art** [thliebig/openEMS-Project#213] [Q&A Usage]
![47ba0a7a9c4f4f1c80f6fc1bac2a105](https://github.com/thliebig/openEMS-Project/assets/144996625/eabf21ed-2ed5-4e6a-90e1-fdd4079a73d0)
![958fa791bc15087c687712ccc87fba7](https://github.com/thliebig/openEMS-Project/assets/144996625/c496d8d1-0e1a-4726-89de-36f6dd233663)
![724b7a5357ebe120241a64eefed8227](https://github.com/thliebig/openEMS-Project/assets/144996625/dae21db8-7bef-47c2-95e7-345839aa0bd6)
Could you please let me know if the error is caused by incorrect environment variable configuration or something else?

**Comment by VolkerMuehlhaus** [comment]
I think your path must be set in **OPENEMS_INSTALL_PATH** variable.

**Reply by Derek-art** [reply]
Thank you for your suggestion. The issue has been resolved, but I've encountered a new problem while running the program. The specific issue is shown in the following image. This is a simulation of the RCS of a metallic sphere from the tutorial. However, why are there extra positional arguments? The tutorial is written this way as well. Please help. Thank you.

![503f95479dd9b292a15f06373595768](https://github.com/thliebig/openEMS-Project/assets/144996625/bfe9b229-f5a2-44ac-a233-cb228d8685fe)
![38d2305a71005ffc68a0e662751819d](https://github.com/thliebig/openEMS-Project/assets/144996625/db9be153-07b4-48df-a4be-bb7dae4f3261)

**Reply by LubomirJagos42** [reply]
hmmm this is from running python script maybe it helps, seems like what you send is right, maybe check import:
```python
from openEMS import openEMS
```

```python
## setup FDTD parameter & excitation function
max_timesteps = 1000000
min_decrement = 0.001 # 10*log10(min_decrement) dB  (i.e. 1E-5 means -50 dB)
CSX = CSXCAD.ContinuousStructure()
FDTD = openEMS(NrTS=max_timesteps, EndCriteria=min_decrement)
FDTD.SetCSX(CSX)
```
